target/linux/pistachio/creator-platform-cascoda-default.config

```diff
@@ -1,8 +1,18 @@
 CONFIG_TARGET_pistachio=y
 CONFIG_TARGET_pistachio_marduk=y
 CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y
+CONFIG_BUSYBOX_CUSTOM=y
+CONFIG_BUSYBOX_CONFIG_FEATURE_WGET_OPENSSL=y
 CONFIG_IMAGEOPT=y
 CONFIG_LOCALMIRROR="http://downloads.creatordev.io/pistachio/marduk/dl"
+CONFIG_OPENSSL_WITH_EC=y
+CONFIG_PACKAGE_ca-certificates=y
+CONFIG_PACKAGE_libopenssl=y
+CONFIG_PACKAGE_openssl-util=y
+CONFIG_PACKAGE_opkg-smime=y
+# CONFIG_PACKAGE_usign is not set
+CONFIG_PACKAGE_zlib=y
+# CONFIG_SIGNED_PACKAGES is not set
```

Comment:
After doing make menuconfig, save and exit during openwrt build, these options get enabled by default.
So we can either have these lines here or we will have to add one extra build step where user will have to disable these options manually.